### ISSUE TITLE
k2s image reset-win-storage does not work correctly

### ISSUE THREAD

**Issue by krotz-dieter** [Siemens-Healthineers/K2s#607]
**Describe the bug**
```
C:\k2s>k2s image reset-win-storage --containerd C:\containerd --max-retry 5 --force-zap
 WARNING  You have not installed K2s setup yet, please start the installation with command 'k2s.exe install' first
```
k2s image reset-win-storage should work without cluster running since script is needing the cluster stopped.

**To Reproduce**
1. Uninstall you cluster if installed
2. call `k2s image reset-win-storage --containerd C:\containerd --max-retry 5 --force-zap`

**Expected behavior**
Storage directory can be cleaned up, cluster should not be necessary.

With this issue also a test case shall be added which checks the behavour.

**Comment by abhitoshj** [comment]
Fixed with #611 on release branch release-1.1.0
Fixed with #618  on main

**Comment by abhitoshj** [comment]
[fix(607): remove uninstall check from reset-win-storage command](https://github.com/Siemens-Healthineers/K2s/pull/618/commits/1631639f3f318157fc59b8a5a5be97c4834233b9)
